portafly/README.md

```diff
@@ -27,3 +27,22 @@ The build is minified and the filenames include the hashes.
 Your app is ready to be deployed!
 
 See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+## Environment Variables
+
+There are some environment variables that we need and we are not happy with React resolution of including them in version contol.
+One could decide to have a local `.env` / `.env.development` files or defining them in `.zshrc` / `.bashrc` or before  running the commands. 
```

Comment:
This is not really true. The purpose of `.env` is to define vars for the project instead of declaring them in the JS. **It's not for env vars** like we know them. For that reason I would not explain this... We can do it when we actually use them.
In the meantime, for vars like PORT and NODE_ENV, they should be declared as part of `yarn start`.

Reply:
I need to  rephrase as follows:
```
There are some environment variables that we need and we are not happy with Create React App resolution of including `.env` files in version contol.
...
```
And in the case of `NODE_ENV`, CRA prevents to override it manually in order to prevent from accidentally deploying a slow development build to production.